### ISSUE TITLE
[linkedin-tool] Better `scroll-margin-top` selector.

### ISSUE THREAD

**Issue by nexushoratio** [nexushoratio/userscripts#4]
By default, using the `scrollIntoView()` API will position the post to the top of the viewport.  Which means the first bit of the post (the user, menu, etc) are hidden under the fixed navbar.  In searching, it appears the way to deal with this is to use the `scroll-margin-top` css property.

Ideally it would be assigned to a specific element which is the *scroll container*.  But I couldn't figure out which one that is.  So to start with, just using an overly broad css rule of `div { scroll-margin-top: VALUEpx }`.  Should make that selector a bit more precise.

**Comment by nexushoratio** [comment]
After working on both the help screen and supporting multiple feeds, I had begun ton wonder if a better selector was possible.

But I just checked, and all current pages that I might want to support all seem to put the interesting content inside the `<main>` element.  So that seems like a good candidate.

**Comment by nexushoratio** [comment]
When working on supporting comments, I realized that `div` isn't a sufficient tag to use as a selector.  And rather than trying to figure out what a good selector might be that wouldn't break other things, I just added the value directly to the comment elements.  After all, we were touching them directly anyway by adding the `tabindex` and calling `focus()`.

So, let's just do that for posts as well.
